java/src/main/java/vowpalWabbit/learner/VWLearnerBase.java

```diff
@@ -17,7 +17,7 @@
  *
  * @author deak
  */
-abstract class VWLearnerBase<T> extends VWBase implements VWTypedLearner<T> {
+public abstract class VWLearnerBase<T> extends VWBase implements VWTypedLearner<T> {
```

Comment:
This is a bad idea I think.  `VWTypedLearner<T>` has everything we need: `predict` and `learn` methods and `close`.  When calling `VWLearners.create`,  just do `VWLearners.create<Object>` or something.